docs/platform/longview/what-is-the-linode-longview-app-for-apache/index.md

```diff
@@ -0,0 +1,362 @@
+
+[![Longview has detected Apache running on this server but was unable to access the server status page. Would you like to attempt to automatically configure mod\_status? This will require reloading Apache to enable. Autoconfigure Mod\_Status: \<Yes\> \<No\>](1451-longview_apache_popup_crop.png)](1451-longview_apache_popup_crop.png)
+
+### Manual Configuration (All Distributions)
+
+To enable the Apache Longview app manually, follow these steps on your Linode via SSH:
+
+1. [SSH into your Linode](/docs/getting-started/#connect-to-your-linode-via-ssh) whose system you are monitoring with Longview.
+
+1.  Make sure **mod\_status** is enabled for Apache (it should be by default). You can follow the instructions list in the [apache.org](https://httpd.apache.org/docs/2.4/mod/mod_status.html) site. Or, on Debian and Ubuntu systems, run this command:
+
+        sudo a2enmod status
+
+1. Update your Apache configuration file to include the block in the example file below. Depending on your Linux distribution and version, your Apache configuration file may be stored in one of the following locations:
+
+    - /etc/apache2/httpd.conf
+    - /etc/apache2/apache2.conf
+    - /etc/httpd/httpd.conf
+    - /etc/httpd/conf/httpd.conf
+
+    {{< file "httpd.conf or apache2.conf" apache >}}
+<IfModule mod_status.c>
+    ExtendedStatus On
+    <Location /server-status>
+        SetHandler server-status
+        Require local
+    </Location>
+</IfModule>
+
+{{< /file >}}
+
+1.  Edit `/etc/linode/longview.d/Apache.conf` to look like the following:
+
+    {{< file "/etc/linode/longview.d/Apache.conf" apache >}}
+location http://127.0.0.1/server-status?auto
+
+{{< /file >}}
+
+1.  Restart Apache:
+
+    >**Debian and Ubuntu**:
+    >
+    >     sudo systemctl restart apache2
+
+    >**Other Distributions**:
+    >
+    >     sudo apachectl restart
+
+1.  Restart Longview:
+
+    > **CentOS, Debian, and Ubuntu**
+    >
+    >     sudo systemctl restart longview
+
+    >  **Other Distributions**
+    >
+    >     sudo service longview restart
+
+1.  Refresh the Longview in your Linode Cloud Manager.
+
+You should now be able to see Longview data for Apache. If that's not the case, proceed to the [Troubleshooting](#troubleshooting) section at the end of this article.
+
+## Viewing Statistics
+
+To see the output for the Longview Apache App:
+
+1.  Log in to the [Linode Cloud Manager](https://cloud.linode.com/).
+2.  Select the **Longview** link in the sidebar.
+3.  Select the **Apache** tab.
+
+    ![The Longview Apache App.](longview_apache_stats.png)
+
+    You'll see the current version of Apache listed on the upper left hand side of the page.
+
+    Mouse over a data point to see the exact numbers for that time. With [Longview Pro](docs/platform/longview/pricing/), you can view older time periods for your data. The next sections cover the Longview Apache App in detail.
```

Comment:
I think this needs a / at the beginning of the the link

Reply:
done